src/build-scripts/build_opencolorio.bash

```diff
@@ -20,13 +20,26 @@ OPENCOLORIO_BUILD_DIR=${OPENCOLORIO_BUILD_DIR:=${LOCAL_DEPS_DIR}/OpenColorIO-bui
 OPENCOLORIO_INSTALL_DIR=${OPENCOLORIO_INSTALL_DIR:=${LOCAL_DEPS_DIR}/dist}
 OPENCOLORIO_BUILD_TYPE=${OPENCOLORIO_BUILD_TYPE:=Release}
 OPENCOLORIO_CXX_FLAGS=${OPENCOLORIO_CXX_FLAGS:="-Wno-unused-function -Wno-deprecated-declarations -Wno-cast-qual -Wno-write-strings"}
+OPENCOLORIO_YAML_CXX_FLAGS=${OPENCOLORIO_YAML_CXX_FLAGS:=''}
+if [[ "$OSTYPE" != "msys" ]]; then
+    OPENCOLORIO_CXX_FLAGS=${OPENCOLORIO_CXX_FLAGS:="-Wno-unused-function -Wno-deprecated-declarations -Wno-cast-qual -Wno-write-strings"}
+    OPENCOLORIO_YAML_CXX_FLAGS=${OPENCOLORIO_YAML_CXX_FLAGS:=''}
+else
+    if [[ "${OPENCOLORIO_BUILD_SHARED_LIBS}" != "ON" ]]; then
+        OPENCOLORIO_CXX_FLAGS=${OPENCOLORIO_CXX_FLAGS:="//MT"}
```

Comment:
`/MT` won't be needed if we don't build wheels for Python 2.7.

Reply:
There's no reason to support Python 2.7. Or any particularly old dependencies. I view the set of users who want python wheels as distinct from the studio users who are stuck on old versions of things. If the "pip install openimageio" only supports fairly modern versions of everything, I have no objections.